app/components/wds-global-navigation.js

```diff
@@ -1,28 +1,38 @@
-import { empty, equal } from '@ember/object/computed';
+import { equal } from '@ember/object/computed';
 import { inject as service } from '@ember/service';
 import { assert } from '@ember/debug';
+import { computed } from '@ember/object';
+import { addQueryParams } from '../utils/url';
 
 import Component from '@ember/component';
 import track from '../utils/wds-track';
 
 export default Component.extend({
 	fetch: service(),
+	fastboot: service(),
+
 	wdsOnSiteNotifications: service(),
 
 	classNames: ['wds-global-navigation'],
 	classNameBindings: [
 		'searchIsActive:wds-search-is-active',
-		'searchIsAlwaysVisible:wds-search-is-always-visible',
 		'model.partner-slot:wds-has-partner-slot',
 		'currentModal:wds-is-modal-opened',
 	],
 
 	searchIsActive: false,
 
-	searchIsAlwaysVisible: empty('model.fandom_overview'),
-
 	isSearchModalOpen: equal('currentModal', 'search'),
 	isUserModalOpen: equal('currentModal', 'user'),
+
+	signinUrl: computed('model.anon.signin', 'fastboot.isFastBoot', function() {
```

Comment:
This computed prop does not use 'fastboot.isFastBoot' -> is adding it to dependencies here intentional?